server/app.mjs

```diff
@@ -401,6 +404,41 @@ const initGPTScriptConfig = async (gptscript) => {
     });
 }
 
+async function updateGPTScriptConfig(gptscript) {
```

Comment:
nit: would be nice if we could just have one idempotent function for initializing/updating the gptscript config.

Reply:
Yeah. I had some ideas about why it might make sense to tweak the logic a bit in the update function, but I changed my mind lol. Combining them into one.